packages/parrot-devtools/package-lock.json

```diff
@@ -270,12 +270,6 @@
 							"dev": true,
 							"optional": true
 						},
-						"ini": {
-							"version": "1.3.5",
-							"bundled": true,
-							"dev": true,
-							"optional": true
-						},
```

Comment:
Why was it deleted instead of updated here?

Reply:
> Why was it deleted instead of updated here?

I believe it's just de-duping